src/components/ListPagination.tsx

```diff
@@ -22,17 +24,25 @@ export const ListPagination = ({
   onPageChange,
   onRowsPerPageChange,
   showPaginationTop,
+  sortingComponent,
 }: ListPaginationProps) => {
+  const itemsStart = count > 0 ? (page - 1) * rowsPerPage + 1 : 0;
+  const itemsEnd = Math.min(page * rowsPerPage, count);
+  const pageCounter = () => <ListPaginationCounter start={itemsStart} end={itemsEnd} total={count} />;
```

Comment:
trenger ikke wrappe dette i en egen funksjon
```suggestion
  const pageCounterComponent = <ListPaginationCounter start={itemsStart} end={itemsEnd} total={count} />;
```
Så kan den brukes sånn:
```
 pageCounterComponent={pageCounterComponent}
```